docs/source/guides/time_series.ipynb

```diff
@@ -0,0 +1,278 @@
+    "\n",
+    "This concern also exists for other primitives that reference earlier values in the dataframe. Because of this, when using primitives for time series feature engineering, one must be incredibly careful to not use primitives on the target column that incorporate the current observation when calculating a feature value."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ed6cc722",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "rolling_mean_primitive = RollingMean(window_length=window_length, \n",
+    "                                     gap=gap,\n",
+    "                                     min_periods=window_length)\n",
+    "\n",
+    "rolling_min_primitive = RollingMin(window_length=window_length, \n",
+    "                                     gap=gap,\n",
+    "                                     min_periods=window_length)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1eb2a6e1",
+   "metadata": {},
+   "source": [
+    "The full list of rolling transform primitives can be seen [here](https://featuretools.alteryx.com/en/latest/api_reference.html#rolling-transform-primitives).\n",
+    "\n",
+    "## Run DFS\n",
+    "\n",
+    "Now that we've definied our time series primitives, we can pass them into DFS and get our feature matrix! \n",
+    "\n",
+    "Let's take a look at an actual feature engineering window as we defined with `gap` and `window_length` above. Below is an example of how we can extract many features using the same feature engineering window without exposing our target value.\n",
+    "\n",
+    "![FE Window](../_static/images/window_calculations.png)\n",
+    "\n",
+    "With the image above, we see how all of our defined primitives get used to create many features from just the two columns we have access to."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "42f52b73",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fm, f = ft.dfs(entityset=es,\n",
+    "               target_dataframe_name='temperatures',\n",
+    "               trans_primitives = (datetime_primitives + \n",
+    "                                  delaying_primitives + \n",
+    "                                  [rolling_mean_primitive, rolling_min_primitive]),\n",
+    "               cutoff_time=pd.Timestamp('1987-1-30')\n",
+    "              )\n",
+    "\n",
+    "f"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9e8ce29d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fm.iloc[:,[0,2, 6, 7, 8, 9]].head(15)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b984ff57",
+   "metadata": {},
+   "source": [
+    "Above is our time series feature matrix! The rolling and delayed features are built from our target column, but do not expose it. We can now use the feature matrix for model building."
```

Comment:
We should a sentence about what the machine learning model could do (predicting daily minimum temperatures?):
```suggestion
    "Above is our time series feature matrix! The rolling and delayed features are built from our target column, but do not expose it. We can now use the feature matrix to create a machine learning model..."
```

Reply:
changed this to

> We can now use the feature matrix to create a machine learning model that predicts future minimum daily temperatures.